ipt_connect/IPTdev/views.py

```diff
@@ -294,10 +294,10 @@ def rounds(request):
 		for team, point in zip(finalteams, finalpoints):
 			finalranking.append([team, point])
 
-		return render(request, 'IPT%s/rounds.html' % params.app_version, {'orderedroundsperroom': orderedroundsperroom, 'finalrounds': finalrounds, "finalranking": finalranking, 'params': params, 'pfs': pfs})
+		return render(request, 'IPT%s/rounds.html' % params.app_version, {'orderedroundsperroom': orderedroundsperroom, 'finalrounds': finalrounds, "finalranking": finalranking, 'params': params, 'pfs': selective_fights})
 
 	else :
-		return render(request, 'IPT%s/rounds.html' % params.app_version, {'orderedroundsperroom': orderedroundsperroom, 'params': params, 'pfs': pfs})
+		return render(request, 'IPT%s/rounds.html' % params.app_version, {'orderedroundsperroom': orderedroundsperroom, 'params': params, 'pfs': selective_fights})
```

Comment:
now that you refactored the "pfs" variable into "selective_fights", you might want to propagate this new name also in the render and associated templates (i.e. replace pfs by selective_fights everywhere)

Reply:
@vbonvin sounds sane, so opened #120